### PR TITLE
cli: add confirmation prompt to closeallchannels

### PR DESCRIPTION
**Pull request by guggero**
Fixes https://github.com/lightningnetwork/lnd/issues/6743.

### CODE REVIEW

File: cmd/lncli/commands.go [hunk 1]
```diff
@@ -1012,6 +1017,11 @@ func closeAllChannels(ctx *cli.Context) error {
 		return err
 	}
 
+	prompt := "Do you really want to close ALL channels? (yes/no): "
+	if !ctx.Bool("skip_confirmation") && !promptForConfirmation(prompt) {
```
Comment (hieblmi) [inline cmd/lncli/commands.go:1021]: ✅

**Review by hieblmi — APPROVED** [review]
Thanks for the fix! LGTM 💾

**Review by bitromortac — APPROVED** [review]
LGTM :four_leaf_clover:

File: cmd/lncli/commands.go [hunk 2]
```diff
@@ -1012,6 +1017,11 @@ func closeAllChannels(ctx *cli.Context) error {
...
+		return fmt.Errorf("action aborted by user")
```
Comment (Chinwendu20) [inline cmd/lncli/commands.go:1022]: Is there any reason you used fmt.Errorf here instead of errors.New since nothing is being formatted?

**Review by ellemouton — APPROVED** [review]
tACK, LGTM 🔬

File: cmd/lncli/commands.go (hunk 2 above)
Comment (guggero) [reply cmd/lncli/commands.go]: No specific reason, mostly just habit. But you're right, makes more sense. Changed.

**Pull request merged**